### ISSUE TITLE
Need definition of transpose in Lin Alg book.

### ISSUE THREAD

**Issue by jbunn3** [TeamBasedInquiryLearning/library#132]
I think it should go in ON2

**Comment by fragandi** [comment]
For vectors, it might fit in ON1 if we say that v.w is actually v^T * w, where * is the usual matrix multiplication. Then in ON2 we define it for sets of vectors/matrix

**Comment by jbunn3** [comment]
>For vectors, it might fit in ON1 if we say that v.w is actually v^T * w, where * is the usual matrix multiplication. 

Good point. That's exactly what I did when I taught the course in the Spring. Steven pointed out that transposes are mentioned in the first definition in the book: [Definition 1.1.1](https://tbil.org/linear-algebra/preview/instructor/LE1.html#LE1-3-2). So we could reference that definition when we do that. 

> Then in ON2 we define it for sets of vectors/matrix

Yes.
